README.md

```diff
@@ -21,18 +21,40 @@ Pending tasks:
 
 - [x] Define YAML schema to represent traits/specifications.
 - [x] Auto-generate Python classes from YAML
-- [ ] Auto-generate CPP and C classes from YAML.
+- [x] Auto-generate CPP classes from YAML
+- [ ] Auto-generate C classes from YAML.
 - [ ] Extend library to cover common post-production entities and
       locales.
 
 ## Installation
 
+### Python
+
 ```shell
 python -m pip install openassetio-mediacreation
 ```
 
+### C++
+
+> **Note**
+>
+> During the configure process, `openassetio-traitgen` will be used to
+> generate C++ trait files. This is assumed to be available in your
+> environment. You may install it via `pip install openassetio-traitgen`
```

Comment:
[fix] For consistency
```suggestion
> environment. You may install it via `python -m pip install openassetio-traitgen`
```

Reply:
https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/25/commits/8fc3f970e046736c897d5c9138abe1688d1da620